### DISCUSSION TITLE
Send text on AutoRelog

### DISCUSSION THREAD

**Discussion by brendan** [ORelio/Minecraft-Console-Client#2002] [Q&A] [answered]
found the answer

**Comment by ORelio** [comment, marked as answer]
> My server is a bungee server, meaning it's a hub server and you can run a command to queue into the server I need, meaning I run a cmd every time I launch the program. Is there a way so on connect & AutoRelog it'll send that cmd for me so AutoRelog actually works?

Answered in #2000
